src/index.ts

```diff
@@ -0,0 +1,123 @@
+import http from 'http'
+import https from 'https'
+import { program } from 'commander'
+
+program
+  .description('A node in the network')
+  .option('-p, --port <port>', 'Port to listen to')
+
+program.parse();
+
+const options = program.opts()
+const PORT = options.port as number || 10309
+
+const requestHandler = (request: http.IncomingMessage, response: http.ServerResponse) => {
+  console.log('I GOT SOMETHING!!!!')
```

Comment:
Lets remove this console.log -> can replace it with something along the lines of "request received"

Reply:
Removed